packages/@okta/vuepress-theme-prose/assets/css/components/_breadcrumb.scss

```diff
@@ -1,52 +1,80 @@
 .breadcrumb {
-  padding: 0.6rem 1.5rem;
-  background-color: cv('gray', '000');
-  border-bottom: $border-default;
+  padding: 0 0 20px;
 
   &--container {
     padding: 0px;
 
     @include media(">=tablet") {
-      padding: 0rem 1.5rem;
+      padding: 0;
     }
 
     ol {
       list-style: none;
       display: flex;
       align-items: center;
       padding: 0em;
+      flex-wrap: wrap;
 
       li {
         display: flex;
         align-items: center;
         margin: 0em 1em 0em 0em;
+        padding: 9px 0;
+
+        @include media("<=desktop") {
+          margin-right: 7px;
+        }
 
         a, span {
           text-decoration: none;
-          color: cv('gray', '800');
-          font-size: 0.8em;
+          color: cv('text', 'darker');
+          font-size: 12px;
+          font-weight: 500;
           padding-right: 1em;
-        }
 
+          @include media("<=desktop") {
+            padding-right: 7px;
+          }
+          @include media("<=tablet") {
+            font-size: 14px;
+            color: cv('text');
+          }
+        }
         svg {
-          fill: cv('gray', '400');
-          width: 0.5em;
-
+          path {
+            @include media("<=tablet") {
+              stroke: cv('text');
+            }
+          }
+        }
+        &:nth-last-child(2) {
+          svg {
+            display: none;
+          }
+        }
+        &:nth-last-child(2) {
+          margin-right: 0;
         }
       }
 
     }
 
     .crumb-show-contents {
-      font-size: 0.8em;
+      font-size: 14px;
       color: cv('action');
       cursor: pointer;
-
-      @include media('>=tablet') {
-        display: none;
-      }
+      display: none;
```

Comment:
seems this button staying none visible on all screens

Reply:
+1, `Show`/`Hide contents` should be shown when tree nav is collapsed

Reply:
It is not in the design. And in tablet and less version was moved to main nav

Reply:
<img width="338" alt="Screenshot at Jul 30 12-29-27" src="https://user-images.githubusercontent.com/71866734/127632921-7ee6c705-9c2a-4624-961f-d5698d88cba7.png">

Reply:
Yep, you're right, it is in design, but as the contents moved to the nav it is redundant

Reply:
sure I agree, but because it's on figma design it's better to discuss this option with designers

Reply:
@alexdahl-okta ^

Reply:
Hmm you may be right @ulyanash, we should follow up with Alex C and Christine. I didn't know we were moving the tree nav into the hamburger.